bucketlist_app/helper.py

```diff
@@ -0,0 +1,49 @@
+def get_or_none(classmodel, **kwargs):
+    try:
+        return classmodel.objects.get(**kwargs)
+    except classmodel.DoesNotExist:
+        return None
+
```

Comment:
https://docs.djangoproject.com/en/1.11/topics/http/shortcuts/#get-object-or-404

from django.shortcuts import get_object_or_404
```
def my_view(request):
    my_object = get_object_or_404(MyModel, pk=1)
```

Is there any difference between this and the custom function?

Reply:
I don't really want to raise an error, I just want to have a null value that's all. But I think I could handle that in my view to do what I want, the reason I didn't is because I'll be repeating `try...except` stuffs.